C7/Map/Civ3TerrainTileset.cs

```diff
@@ -0,0 +1,103 @@
+using Godot;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace C7.Map {
+
+	class TerrainPcx {
+		private static Random prng = new Random();
```

Comment:
I think we have a "master" RNG somewhere in the game data so that everything can be reproduced from the same seed. If that's not sensible to import here, let's put it somewhere that is.

Reply:
We should probably have a settings struct that's easy to initialise with default values instead of injecting a GameData into everything. This RNG is purely cosmetic at the moment as it just determines which terrain texture variant to use